workshops/views.py

```diff
@@ -441,11 +442,33 @@ class EventCreate(LoginRequiredMixin, CreateViewContext):
     model = Event
     fields = '__all__'
 
+@login_required
+def event_edit(request, event_ident):
+    event_form_factory = modelformset_factory(Event, fields='__all__', extra=0)
+    task_form_factory = inlineformset_factory(Event, Task, fields='__all__', extra=1, can_delete=False)
```

Comment:
Could do. The one line solution just seemed more elegant to me. Particularly inlineformset seemed to be the only way to generate a model form that uses foreign keys automatically referring to the event on the page.